app/controllers/courses_controller.rb

```diff
@@ -12,21 +18,43 @@ def show
   end
 
   def update
+    if !current_user.admin?
```

Comment:
Yeah, there are gems for roles, but I think we could just add a `check_admin` method to `application_controller.rb` and add that in a `before_action` for these methods